listenbrainz/webserver/static/css/brainzplayer.less

```diff
@@ -1,143 +1,269 @@
-@primary: #08949b;
-@secondary: #0d696e;
-@controlsHeight: 60px;
-@player-min-height: 320px;
+@listenbrainz-blue: #353070;
```

Comment:
We should probably move colors to colors.less and import from there. Would be convenient to define themes at one place then.

Reply:
I would normally 100% agree, but in this case if we're preparing to make BrainzPlayer an independent package we should reduce dependencies to other parts of Listenbrainz as much as possible.

And when we separate BrainzPlayer, we can replace some of these hardcoded values with a dependency on the design-system package